docs/topics.rst

```diff
@@ -31,6 +31,7 @@ Naming and terminology
 - |:newspaper:| `Common words and how to identify them </blog/newsletter-june-2020/#common-words-and-how-to-identify-them>`__
 - |:movie_camera:| `Even Naming This Talk Is Hard </videos/na/2017/even-naming-this-talk-is-hard-ruthie-bendor/>`__
 - |:movie_camera:| `Whatchamacallit: Controlled Vocabularies for Technical Writers </videos/eu/2015/whatchamacallit-controlled-vocabularies-for-technical-writers-eboillat/>`__
+- 
```

Comment:
Seems to have been added by mistake
```suggestion
```

Reply:
I'm mildly concerned that the YAML linter doesn't pick that up, but yeah, it looks superfluous to me.